docker-compose.yml

```diff
@@ -4,17 +4,17 @@ services:
   postgres_development:
     image: postgres:13.1
     environment:
-      - POSTGRES_USER=user
-      - POSTGRES_PASSWORD=password
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=
```

Comment:
why did you change the env variables here?

Reply:
With the default env variables, I was getting this error,
Error: P1010: User user was denied access to the database DB_DEVELOPMENT_DOCKER.public

And when I used my actual Postgres credentials, it worked fine.

Reply:
interesting, let's see if it happens again in the future (didn't happen for anyone else yet). I wouldn't change it in this PR though, since this is PR is only for tests

Reply:
Yeah sure